src/translation/translation_fr_FR.ts

```diff
@@ -925,7 +925,7 @@
     <message>
         <location filename="../clientdlg.cpp" line="185"/>
         <source>If this LED indicator turns red, the audio stream is interrupted.</source>
-        <translation type="unfinished"></translation>
+        <translation>Si ce voyant devient rouge, le flux audio est interrompu.</translation>
```

Comment:
Why is `If this LED indicator turns red` translated differently here (`filename="../clientdlg.cpp" line="185"`) from above (`filename="../clientdlg.cpp" line="153"`)?

Reply:
`LED` is translated by `DEL` in french. I will try to review all the LED's of the translation.